### PR TITLE
Releasing 2.7.2

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

**Comment by phlax** [comment]
lgtm

**Comment by dwaynebailey** [comment]
lgtm with those minor fixes

File: docs/releases/2.7.2.rst [hunk 1]
```diff
@@ -20,22 +20,29 @@ Changes in Requirements
...
 Below we provide much more detail. These are by no means exhaustive, view the
-`git log <https://github.com/translate/pootle/compare/stable/2.7.0...master>`_
+`git log <https://github.com/translate/pootle/compare/stable/2.7.1...2.7.2>`_
 for complete information.
 
 
 Details of changes
 ==================
 
+- Prevent local TM from crashing if elasticsearch is unavailable. elasticsearch
```
Comment (julen) [inline docs/releases/2.7.2.rst]: It'd be good if `elasticsearch` was properly quoted with backticks — it's hard to read on the live release notes.

Comment (unho) [inline docs/releases/2.7.2.rst]: Merged this change both in `master` and `stable/2.7.2` branches.

Comment (julen) [inline docs/releases/2.7.2.rst]: Thank you @unho!

**Pull request merged**